src/autoscaler/api/quota/quota_management_client.go

```diff
@@ -24,14 +23,14 @@ type Client struct {
 	logger   lager.Logger
 }
 
-func NewClient(config *config.Config, logger lager.Logger, cfClient cf.CFClient) *Client {
+func NewClient(config *config.Config, logger lager.Logger) *Client {
```

Comment:
We're no longer initializing `cfClient` - that would lead to a panic later on.

Reply:
the parameter is never used ... Ive not changed it so it would be happening in the current code too.

Reply:
Right, then we can remove `GetServiceInstancesInOrg`